docs/frameworks/flask.rst

```diff
@@ -35,31 +35,31 @@ Defining manifest
 -----------------
 
 The :mod:`sassutils` defines a concept named :dfn:`manifest`.
-Manifest is building settings of Sass/SCSS.  It specifies some paths
```

Comment:
changing two spaces to one is not a grammatical fix, can you put those back?

Reply:
Shouldn't be a problem. However, I must ask... is it because removing double spaces does not count as a grammatical fix, or are they required by reStructuredText?

Reply:
it's an inconsequential change -- it improves nothing (and in some ways it makes it worse by reducing consistency elsewhere)